src/components/TeaserMyMagazine/TeaserMyMagazine.js

```diff
@@ -51,20 +52,14 @@ const TeaserMyMagazine = ({
       }}
     >
       <section {...css(styles.section)}>
+        <TeaserSectionTitle>Meine Republik</TeaserSectionTitle>
```

Comment:
Let's make this Publikator configurable: use a `title` prop. And forward it in `templates/Front/liveTeasers`—just like `bookmarkLabel`.

While at it please also add the following form config to the `editorOptions` of `type: 'LIVETEASERMYMAGAZINE'`:
```
        form: [
          {
            key: 'title'
          },
          {
            key: 'bookmarkLabel'
          },
          {
            key: 'bookmarksUrl'
          },
          {
            key: 'notificationsLabel'
          },
          {
            key: 'notificationsUrl'
          }
        ]
```
This ensures that those fields are always editable in Publikator. And let's also fix the inconsistency with `bookmarkLabel` and `bookmarksUrl`  🙈

Reply:
done!